### ISSUE TITLE
Use new SW genre values in Argo SW Genre facet

### ISSUE THREAD

**Issue by LynnMcRae** [sul-dlss/argo#644]
Our first take at this turned out to be indexing MODS Genre directly (which is useful), but not what SW uses for its Genre facet. Determine the SW indexing rules and add it to Argo facets.

**Comment by ndushay** [comment]
Note that argo facet would get the same values as SW genre if we unpinned the stanford-mods gem in dor-services:

```
Gem::Specification.new do |s|
  s.add_dependency 'stanford-mods', '= 1.5.3'  # v1.5.4 changes what sw_genre returns, which breaks test expectations.
  # TODO: (stanford-mods) figure out how to keep argo indexed counterparts of searchworks fields in sync w/ current SW behavior.
```

**Comment by ndushay** [comment]
@blalbrit @LynnMcRae   would it be correct to say we would like to change this, but it should be timed with a full reindex of MODS for (all or a subset of) DOR objects?

**Comment by LynnMcRae** [comment]
We were pinning the version of Stanford MODS to before the changes to a new set of values as described in https://github.com/sul-dlss/stanford-mods/issues/66.  We are ready to unpin and move forward.  We will use the current facet to identify "bad" values and progressively reindex affected objects in the background, avoiding the need for a full reindex. This is also fix a related indexing error with a language tag lacking an authority attribute (https://github.com/sul-dlss/dor-services/issues/230)

**Comment by LynnMcRae** [comment]
To update the tests for this, here is where input values from either MODS typeOfResource or MODS genre are mapped into the now confusingly named SW Genre:
https://github.com/sul-dlss/stanford-mods/blob/5a4a552d0cf8b0d6a2e492b6681723f0b359ccad/lib/stanford-mods/searchworks.rb#L330-L349

The only four resulting values are
- Government document
- Conference proceedings
- Technical report
- Thesis/Dissertation

**Comment by cbeer** [comment]
Fixed by 680e9760cc877379fd3a538215c871068adf40a1.
